apps/server/src/modules/board/controller/dto/element/richtext-element.response.ts

```diff
@@ -0,0 +1,38 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { sanitizeRichText } from '@shared/controller';
+import { ContentElementType, InputFormat } from '@shared/domain';
+import { TimestampsResponse } from '../timestamps.response';
+
+export class RichTextElementContent {
+	constructor({ text, inputFormat }: RichTextElementContent) {
+		this.text = sanitizeRichText(text, inputFormat);
```

Comment:
unless I missunderstand something, when we apply sanitisation also on response, means the data coming from db is not trusted. Is not directly wrong, but can be redundant and confusing. At least some kind of comment could explain why we do it

Reply:
Because there might always be an incident that allowed somebody to inject malicous data into our database (import, malfunction, broken sanitasation when storing, ...) . By trusting everything from the database we would endanger any user that malicous code is executed in his/her browser - just by using the schulcloud and looking on courses, boards, etc.